sql_user_check_mysql/README.md

```diff
@@ -0,0 +1,31 @@
+# MYSQL User Check
+
+database_stats.sh script can be used to test connectivity with MYSQL database as well as fetch some stats and metadata. 
+The script supports 2 modes which can be specified with -m flag while running the script. One mode is stats mode which 
+will just print info like list of databases with their size. Count of tables, rows, column, size per schema. 
+Distribution of data by data type. Stats mode is the default mode. Another mode is full_metadata which will print 
+complete information of all columns like their data type, constraints etc.
+
+### Pre-requisites
+
+1. Open or create the ~/.my.cnf file in your user's home directory.
+2. Update the permissions `chmod 600 ~/.my.cnf`
+3. Put your password there like this
+
+```
+[client] 
+password=<your_password>
+```
```

Comment:
Doesn't msql cli already ask for password in their CLI? Why do we need to do this?

Reply:
I think yes, it asks.
But would it be more convenient to do this?? that is not prompt users for password when they run the script.
since we have multiple mysql commands in the script, when user will run the script, they will be prompted multiple times for the password.

Reply:
Okay ask for password yourself and provide it to MySQL commands. This  is too many steps from a user perspective.